src/screens/MainScreen/components/AddCartForm/styles.js

```diff
@@ -0,0 +1,49 @@
+import styled from "styled-components";
+import { themeGet } from "@styled-system/theme-get";
+
+export const Container = styled.div`
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: center;
+  align-items: center;
+`;
+
+export const Title = styled.div`
+  color: ${themeGet("colors.primary")};
+  font-family: ${themeGet("fonts.primary")};
+  font-size: 32px;
+  text-align: center;
+  margin-bottom: 20px;
+`;
+
+export const FormRow = styled.div`
```

Comment:
I think this component could use some padding (see pic in previous comments)

Reply:
Done.